server/index.js

```diff
@@ -7,40 +7,79 @@ const app = express();
 
 app.use(express.static('./public'));
 app.use(cors());
-// Read / GET - read an item
+
+// Read / GET - read a house
 app.get('/houses/*', (req, res) => {
   let houseID = req.path.split('/');
   houseID = houseID[houseID.length - 1];
-  House.findOne({ _id: houseID }, (err, docs) => {
-    res.send(docs);
+  House.getConnection()
```

Comment:
lines 15-31 should be abstracted into a `models` directory. Controllers should be concerned with _selecting_ a routine from an interface, not _providing_ a routine to follow. This router should be able to call something like `Models.getOne(id)` and have it work regardless of what database you're using.